### PR TITLE
Trello board

### PR DESCRIPTION
**Pull request by andrew749**
simple board page which doesn't have pluto logo.

### CODE REVIEW

File: views/display.html [hunk 1]
```diff
@@ -40,7 +40,7 @@
                 </div>
             </div>
             {{/each}}
-        </div>
+            </div>
         <div id="sidebar">
             <div class="widget">
                 <a href="/users">Manage Users</a>
```
Comment (davepagurek) [inline views/display.html:46]: I would add a link in like this one going to /board

File: views/trello.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<html>
+    <div id="trello-board">
+      <iframe id="trello-iframe" width="100%" height="100%" src="https://trello.com/b/VgNPAYxd.html"/>
+    </div>
+    <script type=text/javascript>
+                    window.setInterval("reloadIFrame();", 500);
```
Comment (davepagurek) [inline views/trello.html]: also this is reloading every half second, seems a bit overkill

**Comment by davepagurek** [comment]
If we want the Trello iframe to be embedded in Pluto like this:


...then I'd make trello.html _not_ use `layout: false`. Instead it would be something like this:

``` HTML
<!-- trello.html -->
<div id="trello_container"></div>
{{auto_update "trello_container" "/board/content" 30}}
```

And then have an endpoint for /board/content that _does_ use `layout: false` and only renders the iframe, something like:

``` javascript
// trello_handler.js
pluto.get("/board",function(req,res){
  res.render("trello_content.html",{
      "layout":false
  });
});
```

``` HTML
<!-- trello_content.html -->
<iframe id="trello-iframe" width="100%" height="100%" src="https://trello.com/b/VgNPAYxd.html"/>
```

...so that the iframe content gets pulled in every 30s.

File: views/trello.html (hunk 2 above)
Comment (andrew749) [inline views/trello.html]: oh lol, it was half for testing, it should be like every minute

File: views/display.html (hunk 1 above)
Comment (andrew749) [inline views/display.html:46]: done

File: views/trello.html (hunk 2 above)
Comment (andrew749) [inline views/trello.html]: changed to one minute

**Comment by davepagurek** [comment]
Lgtm :+1:

**Pull request merged**